### ISSUE TITLE
Question about W latent

### ISSUE THREAD

**Issue by Airbus1080** [NVlabs/stylegan2-ada-pytorch#188]
Newbie here. In projection you generate the W latent of the target image by optimizing it but is there a way to generate (the W) without doing projection or starting from Z? It's pretty amazing the model sometimes similarly project a target image that is outside of its domain, I want to take it a step further by directly generating the W latent without projection. I want to know what coherent, recognizable, not blob images it can generate that is outside of the training distribution. Is this possible? Is there a code somewhere out there that can do this?

**Comment by PDillis** [comment]
Depends on the model. For example, FFHQ can generate *any* image as per [Image2StyleGAN](https://arxiv.org/abs/1904.03189). An idea on generating images is to simply generate `w` with the correct shape (e.g., `w = np.repeat(np.random.randn(1, 1, 512), 18, axis=1)` for a 1024x1024 model) and then pass this to `G.synthesis`. Of course, if you wish to explore the latent space outside of W (into W+ as in the paper linked above), then you don't repeat the 'base' dlatent and just let things loose. You will lose coherence though, but you are given more freedom to generate whatever you want.
